generic/tests/cfg/pvpanic.cfg

```diff
@@ -0,0 +1,45 @@
+- pvpanic: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = driver_in_use
+    run_bgstress = pvpanic
+    disable_shutdown = yes
+    monitor_type = qmp
+    event_check = "GUEST_PANICKED"
+    guest_status = "guest-panicked"
+    driver_name = "pvpanic"
+    run_bg_flag = "before_bg_test"
+    timeout = 360
+    RHEL.6:
+        kdump_enable_cmd = "chkconfig kdump on && service kdump restart"
+        kdump_disable_cmd = "chkconfig kdump off && service kdump stop"
+    RHEL.7:
+        kdump_enable_cmd = "chkconfig kdump on && systemctl restart kdump.service"
+        kdump_disable_cmd = "chkconfig kdump off && systemctl stop kdump.service"
+    nmi_enable_cmd = "echo 1 > /proc/sys/kernel/unknown_nmi_panic"
+    nmi_disable_cmd = "echo 0 > /proc/sys/kernel/unknown_nmi_panic"
```

Comment:
You may want to try use `nmi_cmd(you can choose yours)= "echo % > /proc/sys/kernel/unknown_nmi_panic"`.
Then replace the value for cmd later in the if clause.